README.org

```diff
@@ -450,6 +450,36 @@ Use =:context t= to include any prior context in current buffer.
   The day I told you was Wednesday, and the greeting I used was "Ahoy there, me hearty! How be ye today?"
 #+end_src
 
+If you'd like to use multiple contexts instead of a single context you use
```

Comment:
Don't we always use a single context during execution, made up multiple blocks? The context could either use all blocks or a selected fews.

Maybe consider phrasing something along the lines of:

```
If you'd like to cherrypick which blocks are part of a given context, add =:context CONTEXT-NAME= to each block where CONTEXT-NAME is any string. When this form is used only source blocks with same =CONTEXT-NAME= will be included as opposed to every previous block when using =:context t=.
```